common/meta/raft-store/src/state_machine/sm.rs

```diff
@@ -651,19 +678,27 @@ impl StateMachine {
     }
 
     #[tracing::instrument(level = "debug", skip(self))]
-    pub fn get_database(&self, name: &str) -> Result<Option<SeqV<u64>>, ErrorCode> {
-        let dbs = self.databases();
+    pub fn get_database_id(&self, name: &str) -> Result<Option<SeqV<u64>>, ErrorCode> {
+        let dbs = self.database_lookup();
         let x = dbs.get(&name.to_string())?;
         Ok(x)
     }
 
-    pub fn get_databases(&self) -> Result<Vec<(String, u64)>, ErrorCode> {
+    pub fn get_database_by_id(&self, did: &u64) -> Result<Option<SeqV<DatabaseMeta>>, ErrorCode> {
+        let x = self.databases().get(did)?;
+        Ok(x)
+    }
+
+    pub fn get_databases(&self) -> Result<Vec<(String, u64, DatabaseMeta)>, ErrorCode> {
```

Comment:
It would be better to return `Result<Vec<DatabaseInfo>, ErrorCode>`, to keep style consistent with `get_tables() -> Result<Vec<TableInfo>, _>`

Reply:
got it.